Makefile

```diff
@@ -154,6 +154,10 @@ tailscale-fdroid.apk: $(AAR)
 	(cd android && ./gradlew test assembleFdroidDebug)
 	mv android/build/outputs/apk/fdroid/debug/android-fdroid-debug.apk $@
 
+tailscale-new.apk: 
+	(cd android && ./gradlew assembleNewAppDebug)
+	mv ./android/build/outputs/apk/newapp/debug/android-newapp-debug.apk $@
```

Comment:
recommend using `install -C` here (and above) rather than `mv`, as that way you'll get near 0 cost incremental rebuilds if nothing actually changed